.github/workflows/ci.yml

```diff
@@ -423,32 +426,40 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v3
-      - name: Install toolchain
+      - name: Install Rust toolchain
+        if: ${{ !env.ACT }}
```

Comment:
nit: I'd prefer to do the act-related stuff in a separate PR since it's not related to updating the dependencies